php-binance-api.php

```diff
@@ -240,9 +243,9 @@ protected function setupProxyConfigFromFile(string $file = null)
      * @param $flags array addtional options for order type
      * @return array with error message or the order details
      */
-    public function buy(string $symbol, $quantity, $price, string $type = "LIMIT", array $flags = [])
+    public function buy(string $symbol, $quantity, $price, string $type = "LIMIT", array $flags = [], $futures = false)
     {
-        return $this->order("BUY", $symbol, $quantity, $price, $type, $flags);
+        return $this->order("BUY", $symbol, $quantity, $price, $type, $flags, false, $futures);
```

Comment:
Or at least use the `$flags` array to set `'fapi' => true` and send it to the third parameter of `httpRequest()` like it's done with `wapi` and `sapi` already.

Reply:
I'm more for a separate class or at least separate functions like fBuy and/or fHttpRequest or whatever would fit best. Having the wapi and sapi is according to me already more then enough in httpRequest... (personal view!)

Reply:
Yeah, SAPI *could* be a separate class as well to make it cleaner. WAPI is deprecated as of today: https://www.binance.com/en/support/announcement/f45dde7da58b473aa885349946bed269